ci/test_notebooks.sh

```diff
@@ -29,7 +29,7 @@ NBTEST="$(realpath "$(dirname "$0")/utils/nbtest.sh")"
 
 # Add notebooks that should be skipped here
 # (space-separated list of filenames without paths)
-SKIPNBS="binary_predicates.ipynb cuproj_benchmark.ipynb"
+SKIPNBS="binary_predicates.ipynb cuproj_benchmark.ipynb ZipCodes_Stops_PiP_cuSpatial.ipynb"
```

Comment:
proposing skipping this until https://github.com/rapidsai/cuspatial/issues/1426 is resolved

Reply:
Now that the aforementioned issue has been resolved, can we please readd zipcodes?

```suggestion
SKIPNBS="binary_predicates.ipynb cuproj_benchmark.ipynb"
```

Reply:
ah yes! good catch, totally agree.